### ISSUE TITLE
Release docker image for self-hosting

### ISSUE THREAD

**Issue by alexklibisz** [dalibo/explain.dalibo.com#26]
Hi, thanks for building and maintaining this extremely useful tool.

Do you know if there is a docker image of the application that we can just deploy without having to re-build the image? If not, would you be interested in a contribution that makes this possible? Maybe a github workflow that builds and publishes the image to ghcr?

**Comment by alexklibisz** [comment]
I created a repo to publish the server image: https://github.com/alexklibisz/pev2-docker. 

The server gets published here: https://github.com/alexklibisz/pev2-docker/pkgs/container/pev2-server. 

The publish workflow is here: https://github.com/alexklibisz/pev2-docker/blob/main/.github/workflows/publish.yaml

**Comment by pgiraud** [comment]
Sorry for not answering earlier. I'd like to thank you for your interest and for the contribution. I'll have a look whenever possible.

Don't you think that this (the publication workflow) could go into the `explain.dalibo.com` project and be maintained there? Reading your first message, I do think it was your first intention and you were just waiting for my answer.

**Comment by alexklibisz** [comment]
Yeah I think it's best for the publish workflow and docker image to live in this repo. I'll try to make a PR with this later today.

**Comment by alexklibisz** [comment]
PR here: https://github.com/dalibo/explain.dalibo.com/pull/27

**Comment by pgiraud** [comment]
I didn't realize that we had to explicitly set the visibility to public for github packages. After some thoughts I will work on this a bit more to publish images to docker hub instead of github packages. I will work on this again hopefully next week.
cc @joshuataylor

**Comment by pgiraud** [comment]
Docker images are now available in Docker Hub. https://hub.docker.com/r/dalibo/explain.dalibo.com
See `docker-compose.yml` for example usage.
